### DISCUSSION TITLE
Convert V1.2 PCB to V1.3

### DISCUSSION THREAD

**Discussion by Electrotechnique** [ElectroTechnique/electrotechnique.github.io#6] [Hardware Issues]
**Modify V1.2 PCB to V1.3**

 **1**  Soft MIDI Thru/Out - You need to take the front panel off to work on the top side of the PCB. Break the track between the via and pin 1 of the 74HC04 Hex Inverter. Solder a wire between pin 3 (TX) of the Teensy and pin 1 of the 74HC04 Hex Inverter. See the User Manual for how the MIDI Thru/Out setting works.

[![V12-13_Modification](https://electrotechnique.cc/V12-13_Modification.jpg)](https://electrotechnique.cc/V12-13_Modification.jpg)

 **2**  Display backlight control (THIS IS NOT CURRENTLY USED AND INESSENTIAL) - Solder a wire between Teensy 4.1 pin 6 (marked as 8 in the picture below) of the Teensy and the BL (backlight) pin of the display (the pin on the left marked as 8 in the picture below.)

 **3** Display SCL (clock) noise - breaking the SCL track to the display on the bottom of the PCB and soldering a 100 ohm resistor has been found to cut down on some of the noise on the analogue output at high volume. _A small SMD resistor is best_. On the V1.3 PCB, the track has been re-routed slightly and has the resistor.

[![SCK_Fix](https://electrotechnique.cc/SCK_Fix.jpg)](https://electrotechnique.cc/SCK_Fix.jpg)

**Comment by cdw2000** [comment]
I haven't noticed this display noise issue on my Teensy 3.6 board using a Teensy 4.1 conversion board.   Was the routing in that area significantly different on the older board?
